src/Compilers/CSharp/Portable/BoundTree/BoundNodeExtensions.cs

```diff
@@ -65,5 +65,24 @@ public static bool IsConstructorInitializer(this BoundCall call)
             node.WasCompilerGenerated = true;
             return node;
         }
+
+        public static bool ContainsUsingDeclarationStatement(this BoundStatement statement, out BoundUsingLocalDeclarations declaration)
+        {
+            declaration = null;
+            if (statement is BoundUsingLocalDeclarations localDeclaration)
```

Comment:
>is [](start = 26, length = 2)

Consider using `switch (statement.Kind)` rather than `is`.

Reply:
(Removed as unused)